README.md

```diff
@@ -1,45 +1,83 @@
 # MVP III
 
+## Background
+
+Code and instructions for building the 'brain' of the controled environment hydroponics unit.
+It is mostly a collection of python code that runs on a Raspberry Pi (or similar device). See the OpenAg [forums](http://forum.openag.media.mit.edu/) for discussion and issues:
+
+The MVP (Minimal Viable Product) is a simplified version of the MIT OpenAg Food Computer.
+
+## Installation
+
+- ### Set up the Raspberry Pi with the NOOBS installation
+
+  - You can use the instructions in the official Rasbperry Pi [quickstart guide](https://www.raspberrypi.org/learning/software-guide/quickstart/) under 'Install Raspbian with NOOBS'
```

Comment:
I updated this part to refer to the raspberry pi docs rather than trying to re-create them. The quickstart guide is really great in detailing the NOOBS installation